src/main/java/core/basesyntax/service/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class RegistrationException extends Exception {

```

Comment:
You should create your custom unchecked exception and throw it in case of invalid data.

you create the checked exception, be attentive!